enhancements/network/upi-node-ip-selection.md

```diff
@@ -0,0 +1,213 @@
+---
+title: upi-node-ip-selection
+authors:
+  - "@pliu"
+reviewers: 
+  - "@racedo"
+  - "@makentenza"
+  - "@cybertron"
+  - "@mkowalski"
+approvers:
+  - "@danwinship"
+  - "@trozet"
+api-approvers: 
+  - "None"
+creation-date: 2023-02-08
+last-updated: 2023-02-08
+tracking-link:
+  - https://issues.redhat.com/browse/NP-687
+see-also:
+  - "/enhancements/network/ip-interface-selection"
+replaces:
+superseded-by:
+---
+
+# Node IP Selection UPI
+
+## Summary
+
+When Openshift is deployed on user-provisioned infrastructure, and there are
+more than one NIC available on the hosts. We need a reliable way to guarantee
+all the openshift components can use the correct IP/interface.
+
+## Motivation
+
+Today, we do not run the nodeip-configuration service on vSphere UPI. It means
```

Comment:
@cybertron @pliurh is this something other than what is being referenced here?
https://github.com/openshift/machine-config-operator/pull/3460

Reply:
That's part of it. I think the other part is exposing it as an install-config parameter so users don't have to write a machine-config to use it.